pinn/networks/pinet2.py

```diff
@@ -183,7 +183,7 @@ def call(self, tensors):
         Returns:
             output (tensor): an updated output tensor with shape `(n_atoms, out_units)`
         """
-        ind_1, p1, p3, prev_output = tensors
```

Comment:
Why did you remove p3 here?

Reply:
To reuse outlayer for other variables, for example, i3, i5, p5 etc. ?

Reply:
Then should p1 be renamed to "prop" or something like in the documentation? To indicate general use?